hypothesis-python/src/hypothesis/strategies.py

```diff
@@ -23,6 +23,7 @@
 import string
 import datetime as dt
 import operator
+from sys import version_info as python_version_info
```

Comment:
Nit: Is there a reason to not just import `sys` and use `sys.version_info`? It's shorter and IMO slightly clearer.